MaxDataDiskCount is lower for Classic RDFE VMs

axelgMS opened the pull request:
CRIs 57005309 - 58844713 - 53789825 ...

Customers with Classic VMs are reporting that MaxDataDiskCount is 2x lower that what's written in the doc. Confirmed in Source Code. They would get errors like the one below if they try to add another Data Disk
Too many data disks specified for virtual machine netserv. The maximum number of data disks currently permitted is 4. The current number of data disks is 4. The operation is attempting to add 1 additional data disks

In order to stop them opening cases / CRIs, we need to update the doc.

azurecla commented:
Hi __@anosov1960__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

axelgMS committed:
MaxDataDiskCount is lower for Classic RDFE VMs

CRIs 57005309 - 58844713 - 53789825 ...

Customers with Classic VMs are reporting that MaxDataDiskCount is 2x lower that what's written in the doc. Confirmed in Source Code. They would get errors like the one below if they try to add another Data Disk
Too many data disks specified for virtual machine netserv. The maximum number of data disks currently permitted is 4. The current number of data disks is 4. The operation is attempting to add 1 additional data disks

In order to stop them opening cases / CRIs, we need to update the doc.

GitHubber17 committed:
Update virtual-machines-common-sizes-general.md
